src/main/java/org/owasp/benchmark/score/parsers/KiuwanReader.java

```diff
@@ -41,15 +41,24 @@ public TestResults parse( File f ) throws Exception {
 //		String resultsFormatVersion = obj.getString( "version" ); // Note: no threadfix version info included in format.
 
 		JSONArray findings = obj.getJSONArray("findings");
+		
+		String source = obj.getString("source");
 
-		TestResults tr = new TestResults( "Kiuwan", true, TestResults.ToolType.SAST);
+		TestResults tr = new TestResults(source, true, TestResults.ToolType.SAST);
 		// Scan time is not included in the threadfix schema. But scan time is provided on their web site next to results
 		tr.setTime(f);  // This grabs the scan time out of the filename, if provided 
 						// e.g., Benchmark_1.2_Kiuwan-660.threadfix, means the scan took 660 seconds.
 
 		// Set the version of Kiuwan used to do the scan (Can't because that info isn't provided)
 		// It is provided on their web site. Looks like: Engine version   master.p561.q11382.a1870.i501
-//		tr.setToolVersion(driver.getString("version"));
+		// We will use the created date. format: "created":"2019-11-05T21:24:49Z"
+		String created = obj.getString("created");
+		if (null != created) {
+			created = created.replace("-", "");
+			created = created.replace(":", "");
+			created = created.trim();
+			tr.setToolVersion(created);
+		}
```

Comment:
I don't like this change. Created date is NOT an engine version number. Isn't the created date, the date the results file was created? If so, please revert this change.

Reply:
Nevermind. It appears you fixed this in your next checkout.